java/query/query-lambda/pom.xml

```diff
@@ -42,6 +42,10 @@
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-lambda-java-events</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-lambda-java-runtime-interface-client</artifactId>
+        </dependency>
```

Comment:
Is it a good idea to put this in the classpath whether we're deploying via Docker or not? We could just put both this and the lambda jar on the classpath when we run the lambda, as in the AWS example Dockerfile:

https://github.com/aws/aws-lambda-java-libs/tree/main/aws-lambda-java-runtime-interface-client